docs/tempo/website/operations/generic_forwarding.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Generic forwarding
+weight: 13
+---
+
+# Generic forwarding
+
+Generic forwarding is a feature that allows asynchronous replication of ingested traces. The distributor will write received spans to both the ingester and defined endpoints if enabled. This feature works in a "best-effort" manner, meaning that no retries happen if an error occurs during replication. 
```

Comment:
Good explanation of the forwarder. I've made a suggestion to tighten the content a little. 

The second sentence says "The distributor"... What does this refer to? We haven't used this term before in this page. 
```suggestion
Generic forwarding allows asynchronous replication of ingested traces. The distributor writes received spans to both the ingester and defined endpoints, if enabled. This feature works in a "best-effort" manner, meaning that no retries happen if an error occurs during replication. 
```

Reply:
The forwarder is a new subcomponent of the distributor. It wasn't there before.

Reply:
Applied your suggestion.